### ISSUE TITLE
Button Disappear

### ISSUE THREAD

**Issue by Wehemey** [APeng215/FiltPick#10]
This mods seems to disable the buttons added to the inventory by other mods
mod file
fabric-api-0.92.2+1.20.1
fabric-language-kotlin-1.11.0+kotlin.2.0.0
FiltPick-fabric-mc1.20.1-1.2.3
quickstack-1.0.2+1.20.1
inventorio-mc1.20.1-fabric-v1.10.3

**Comment by Razor235T** [comment]
Inventorio adds its own inventory. You'll need to enable Toggle Button option it it, then you will be able to swap inventory to vanilla one, where Filtpick button is.

**Comment by Wehemey** [comment]
No, I'm pretty sure it's the mod, even with Inventorio disabled!

**Comment by APeng215** [comment]
> No, I'm pretty sure it's the mod, even with Inventorio disabled!

You are right. This is caused by the high render priority of FiltPick. I may fix this later.

**Comment by APeng215** [comment]
This bug has been fixed at least in version 1.21.5.
